code/processes/gateway.q

```diff
@@ -193,7 +193,9 @@ deleteresult:{[queryid] .gw.results : (queryid,()) _ .gw.results}
 
 // add a result coming back from a server
 addserverresult:{[queryid;results]
- serverid:first exec serverid from .gw.servers where active, handle=.z.w;
+ serverid:$[@[get;`.finspace.enabled;0b];
```

Comment:
```suggestion
 serverid:$[@[value;`.finspace.enabled;0b];
```

conventionally `get` is for file access, for resolving a symbol to variable content use `value`

Reply:
resolved now